app/views/components/pickadate.html

```diff
@@ -4,3 +4,8 @@
   </span>
   <input type="text" placeholder="Select a date..." class="form-control" ng-disabled="disabled">
 </div>
+<div ng-if="(minDate || maxDate) && (minDate != '' || maxDate != '')" class="help-block row" translate>
```

Comment:
I think each element containing text added here needs to have the `translate` attribute added to it. I think it needs to be removed from the parent div and added to each of the child elements since they are not shown or hidden all at once. I'm also not sure `translate` works with html tags in it. https://angular-gettext.rocketeer.be/dev-guide/annotate/